src/lib.rs

```diff
@@ -59,7 +59,10 @@ mod umessage;
 pub use umessage::{UMessage, UMessageBuilder, UMessageError};
 
 mod uri;
-pub use uri::{UUri, UUriError};
+pub use uri::{
+    UUri, UUriError, RESOURCE_ID_MIN_EVENT, RESOURCE_ID_RESPONSE, WILDCARD_AUTHORITY,
```

Comment:
hmm, I see the problem with the `uri` module not being public. However, I do not really like to expose these constants on the top level because then they are missing any context: it is quite unclear what `up_rust::RESOURCE_ID_MIN_EVENT` is useful for, isn't it? Maybe it would be better to make the `uri` module public instead?

Reply:
Agree with you